bee-api/bee-rest-api/src/types/body.rs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use serde::Serialize;
+
+/// Represents the data that can be included into `SuccessBody` and `ErrorBody`.
```

Comment:
I suggest adding the term "marker trait" to make it clear to the user, that its an empty trait. So .... "A marker trait to represent...."